plugins/http_plugin/http_plugin.cpp

```diff
@@ -138,6 +139,13 @@ namespace eosio {
 
          websocket_server_type    server;
 
+         uint16_t                                    thread_pool_size = 2;
```

Comment:
So we still want to default to 2 threads, even though we can have 1?

Reply:
Since we now only load http_plugin if we need it, sure.